src/log.h

```diff
@@ -0,0 +1,93 @@
+#pragma once
+
+#include <iostream>
+#include <sstream>
+
+#include "bpftrace.h"
+
+namespace bpftrace {
+
+// clang-format off
+enum class LogType
+{
+  debug,
+  info,
+  warning,
+  error
+};
```

Comment:
Typically enums are all caps (eg https://github.com/torvalds/linux/blob/master/include/uapi/linux/bpf.h#L122 ). I think that makes the log api look nicer too. So it'd be:
```
LOG(DEBUG) << "whatever";
```
That's also how it looks for glog (google's log library)

Reply:
I'd agree, will change.